scripts/linters/custom_eslint_checks/lib/rules/break-after-parens.js

```diff
@@ -0,0 +1,110 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview There should be a break after parenthesis
+ * @author Oppia
+ */
+'use strict';
+
+// -----------------------------------------------------------------------------
+// Rule Definition
+// -----------------------------------------------------------------------------
+
+module.exports = {
+  meta: {
+    type: 'layout',
+    docs: {
+      description: 'There should be a break after parenthesis',
+      category: 'Stylistic Issues',
+      recommended: false
+    },
+    fixable: null,
+    schema: [],
+    messages: {
+      expectedAfter: 'Expected newline after \'(\'.'
+    }
+  },
+
+  create: function(context) {
+    const sourceCode = context.getSourceCode();
+    const lines = sourceCode.lines;
+
+
+    // ----------------------------------------------------------------------
+    // Helpers
+    // ----------------------------------------------------------------------
+
+
+    // ----------------------------------------------------------------------
+    // Public
+    // ----------------------------------------------------------------------
+
+    return {
+      Program: function checkParenSpaces(node) {
```

Comment:
Can you please mention (here in this thread) all the places where we are asking the user to break-line after parenthesis? (if it's not implemented to handle all the cases.)

I need this to review the later part of this file!

Reply:
We need line break after parenthesis in case of multiline line hanging indentation, except for -- `while`, `if`, `for`, `else if`, any line starting with `angular.module('oppia')`, `it` and `describe`.

Reply:
@bansalnitish, I'm not sure whether we had thought of excluding `it` and `describe`. Can you please confirm?

Reply:
Yes, we did thought of excluding.

Reply:
IIRC, we had finalized to consider `it` and `describe` as a function call and don't consider it as a special case do whatever we do for function calls, just to have consistency style for all the function calls. Anyway, I'm fine with excluding these if you all have decided. Thanks for confirming! :)